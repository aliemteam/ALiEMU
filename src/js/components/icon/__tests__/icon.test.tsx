import React from 'react';
import renderer from 'react-test-renderer';

import Icon from '../';

describe('<Icon />', () => {
    it('should render an accessibility icon', () => {
        const tree = renderer.create(<Icon icon="accessibility" />);
        expect(tree).toMatchSnapshot();
    });
    it('should render an account_circle icon with custom attributes', () => {
        const tree = renderer.create(
            <Icon icon="account_circle" size={50} color="red" />,
        );
        expect(tree).toMatchSnapshot();
    });
});
