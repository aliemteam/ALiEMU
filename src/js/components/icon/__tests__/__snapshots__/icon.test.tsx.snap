// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon /> should render an accessibility icon 1`] = `
<i
  className="material-icons"
  style={
    Object {
      "color": undefined,
      "fontSize": 18,
    }
  }
>
  accessibility
</i>
`;

exports[`<Icon /> should render an account_circle icon with custom attributes 1`] = `
<i
  className="material-icons"
  style={
    Object {
      "color": "red",
      "fontSize": 50,
    }
  }
>
  account_circle
</i>
`;
